src/modules/fundingManager/depositVault/FM_DepositVault_v1.sol

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.23;
+
+// Internal Interfaces
+import {IOrchestrator_v1} from
+    "src/orchestrator/interfaces/IOrchestrator_v1.sol";
+import {IFundingManager_v1} from "@fm/IFundingManager_v1.sol";
+import {IFM_DepositVault_v1} from
+    "@fm/depositVault/interfaces/IFM_DepositVault_v1.sol";
+
+// Internal Dependencies
+import {Module_v1} from "src/modules/base/Module_v1.sol";
+
+import {Initializable} from "@oz-up/proxy/utils/Initializable.sol";
```

Comment:
Unused import

Reply:
Valid, fixes in [this commit](https://github.com/InverterNetwork/contracts/pull/646/commits/8eb10a27e4b749a98bd0f2dde49414cd32244193). PR that addresses all of the fixes is #646. 🙏